documentation/release_notes.markdown

```diff
@@ -19,7 +19,7 @@ This Puppet Server release provides a new workflow and API for certificate issua
 
 ### New features
 
-- There is now a CLI command for setting up the certificate authority, called `puppetserver ca`. See [Puppet Server: Intermediate CA](/puppetserver/latest/intermediate_ca.html) for more information. [(SERVER-2172](https://tickets.puppetlabs.com/browse/SERVER-2172))
```

Comment:
We still might want to link to the intermediate ca docs somewhere? If not a mention about this here, then maybe below in the bug fix section were we talk about intermediate CA support?